src/lib/util/misc.c

```diff
@@ -1185,3 +1187,40 @@ int fr_digest_cmp(uint8_t const *a, uint8_t const *b, size_t length)
 
 	return result;		/* 0 is OK, !0 is !OK, just like memcmp */
 }
+
+/** Create an empty file. (It will overwrite an existing file).
+ *
+ * @param filename path to file.
+ * @param mode Specifies the file mode bits be applied.
+ */
+int fr_file_touch(char const *filename, mode_t mode) {
+	int fd;
+
+	fd = open(filename, O_WRONLY | O_CREAT, mode);
+	if (fd < 0) {
+		fr_strerror_printf("fr_file_touch() Failed creating file \"%s\": %s", filename, fr_syserror(errno));
```

Comment:
Don't really need the function name here, it tends to look messy when they're used in common utility functions